addons/calendar/views/calendar_views.xml

```diff
@@ -132,6 +132,8 @@
                             <field name="start" widget="daterange" options="{'end_date_field': 'stop'}" invisible="allday" required="not allday" readonly="not user_can_edit"/>
                             <field name="stop_date" invisible="1"/>
                             <field name="stop" invisible="1" />
+                            <field name="unavailable_partner_ids" invisible="1" /> <!-- this field will be used in
+                            many2many_attendees widget -->
```

Comment:
You can put everything on one line.

If it's really too long (but it doesn't appear to be), put the whole comment above the field.